README.md

```diff
@@ -0,0 +1,66 @@
+# Z-Way for OpenLuup
```

Comment:
This is an excellent 'how-to' for replacing Vera, but I think it's a separate issue from the ZWay plugin itself.  We need to find a better place to put this.